src/sources/soundsourceffmpeg.cpp

```diff
@@ -454,11 +454,7 @@ QStringList SoundSourceProviderFFmpeg::getSupportedFileExtensions() const {
 SoundSourceProviderPriority SoundSourceProviderFFmpeg::getPriorityHint(
         const QString& supportedFileExtension) const {
     Q_UNUSED(supportedFileExtension)
-    // TODO: Increase priority to Default or even Higher for all
-    // supported and tested file extension?
-    // Currently it is only used as a fallback after all other
-    // SoundSources failed to open a file or are otherwise unavailable.
-    return SoundSourceProviderPriority::Lowest;
+    return SoundSourceProviderPriority::Higher;
```

Comment:
This has already been rejected (limited to M4A/AAC) and would prioritize SoundSourceFFmpeg for MP3/M4A/AAC/ALAC/AIFF/WAV on all platforms.

This would also be ambiguous without raising the priority for OGG/FLAC/OPUS from `Higher` to `Highest` (which we should do anyway).